thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js

```diff
@@ -152,11 +196,21 @@ export default Route.extend({
    * @method model
    * @return {Object}
    */
-  model() {
+  model(transition) {
+    // Get duration data
+    const {
+      duration,
+      startDate,
+      endDate
+    } = transition;
+
     return RSVP.hash({
       // Fetch all alert group configurations
       configGroups: fetch('/thirdeye/entity/ALERT_CONFIG').then(res => res.json()),
-      applications: fetch('/thirdeye/entity/APPLICATION').then(res => res.json())
+      applications: fetch('/thirdeye/entity/APPLICATION').then(res => res.json()),
```

Comment:
TODO (follow-up PR): let's continue the good work abstracting the url strings into a `api`.js

Reply:
Ah - I intended to do that in this PR. Thanks for the reminder.